azure-pipelines.parallel.yml

```diff
@@ -0,0 +1,155 @@
+pr:
+  - master
+
+trigger:
+  - master
+
+variables:
+  - group: fabric-variables
+
+pool:
+  vmImage: 'Ubuntu 16.04'
+
+jobs:
+  - job: Build
+    steps:
+    # TODO: remove, temporarily disabled to speed debugging
+    # - checkout: none
+    #   clean: false
+
+    # TODO: uncomment, temporarily disabled to speed debugging
+    - template: azure-pipelines.tools.yml
+
+    - script: |
+        yarn
+      displayName: yarn
+
+    - script: |
+        npm run checkchange
```

Comment:
nit: `yarn run`

Reply:
This is copied code from the existing pipeline.

Reply:
(I'm not sure why `yarn` wasn't already used for much of this... was it just because of the newer `yarn` dep?)

Reply:
actually do:

```suggestion
        yarn checkchange
```